frontend/src/lib/lemon-ui/LemonCalendar/LemonCalendar.tsx

```diff
@@ -145,6 +165,47 @@ export function LemonCalendar(props: LemonCalendarProps): JSX.Element {
                     </table>
                 )
             })}
+            {showTime && (
+                <div className="LemonCalendar__time flex divide-x border-l">
+                    <ScrollableShadows direction="vertical">
+                        {[12, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11].map((hour) => {
+                            const buttonProps = props.getLemonButtonTimeProps?.({
+                                unit: 'h',
+                                value: hour,
+                            })
+
+                            return (
+                                <LemonButton fullWidth key={hour} {...buttonProps}>
+                                    <span className="w-full text-center px-2">{String(hour).padStart(2, '0')}</span>
+                                </LemonButton>
+                            )
+                        })}
+                        <div className="h-[229px]" />
```

Comment:
It was the height of the div minus the height of a time button so that there was scroll padding to align the last hour and minute to the top of the div. Realised from your comment above that it actually doesn't work when there are more weeks in the month.

New approach calculates the height as 100% and subtracts the button height (which is now a named variable for clarity). Probably the best we're going to do here